packages/api/src/codegen/languages/typescript/index.ts

```diff
@@ -383,7 +382,7 @@ sdk.server('https://eu.api.example.com/v14');`),
           name: 'createSDK',
           initializer: writer => {
             // `ts-morph` doesn't have any way to cleanly create an IIFE.
-            writer.writeLine('(() => { return new SDK(); })()');
+            writer.write('(() => { return new SDK(); })()');
```

Comment:
Changing this from `.writeLine()` to `.write()` fixes a quirk where the ending `;` would get placed on the following line because `.writeLine()` terminates the thing we're writing with a `\n`.